presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPruneWindowColumns.java

```diff
@@ -237,7 +239,9 @@ private static PlanNode buildProjectedWindow(
                                                 UNBOUNDED_PRECEDING,
                                                 Optional.of(startValue2),
                                                 CURRENT_ROW,
-                                                Optional.of(endValue2)))),
+                                                Optional.of(endValue2),
+                                                Optional.empty(),
+                                                Optional.empty()))),
```

Comment:
this is going to fail as it is expected these values to be present when `startValue` and `endValue` are present